js/controller.js

```diff
@@ -122,6 +123,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.subscribe(OO.EVENTS.DESTROY, 'customerUi', _.bind(this.onPlayerDestroy, this));
       this.mb.subscribe(OO.EVENTS.EMBED_CODE_CHANGED, 'customerUi', _.bind(this.onEmbedCodeChanged, this));
       this.mb.subscribe(OO.EVENTS.CONTENT_TREE_FETCHED, 'customerUi', _.bind(this.onContentTreeFetched, this));
+      this.mb.subscribe(OO.EVENTS.STYLE_FETCHED, 'customerUi', _.bind(this.onStyleFetched, this));//xenia: to be replaced by a more appropriate event
```

Comment:
This event listener has to be changed. I am proposing to use CMS-less new event: ASSET_UPDATED